### PR TITLE
initial README.md

### PR DESCRIPTION
**Pull request by Horta**
Hello Catalina,

I was exploring your project a bit and I thought that it would be a great idea to have a README.md showing in the main github page some simple steps to get started into your project.

That is really my first time I contribute to R package so I don't know the standard ways to do stuff. For example, I don't know what is the common way of hosting a documentation explaning your project (that I suppose comes from vignettes/BASiCSIntro.Rmd, so shall we just create a link here pointing to here?).

Anyway, I hope it is a good start from my part and I will come with those handy examples for parallelizing your code tomorrow =)

Best,
Danilo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+```R
+# install.packages("devtools")
+# source("https://bioconductor.org/biocLite.R")
+# biocLite("BiocGenerics")
+devtools::install_github("catavallejos/BASiCS")
+```
+
+## Quick start
+
+TODO: a quick start for BASiCS. Like vignette("some-stuff")
```
Comment (Horta) [inline README.md]: Watch out this part here. There is a TODO =P

File: README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Cell-specific normalization constants are estimated as part of the model parameters.
+- Technical variability is quantified based on spike-in genes that are artificially introduced to each analysed cells lysate.
+- The total variability of the expression counts is decomposed into technical and biological components.
+
+## Installation
+
+Get the released version from CRAN:
+
+```R
+install.packages("BASiCS")
```
Comment (Horta) [inline README.md]: Looks like BASiCS is not on CRAN. Is there a burocraticy process to fullfill before having a package available there?

**Comment by catavallejos** [comment]
Hi Danilo,

Thank you for going through the code! 

I agree, having a README file is a good idea, although for examples I'd just point towards the vignette. I'll add that now. 

About CRAN. Indeed, BASiCS is not there. Actually, the idea is to submit it to BioConductor which has more visibility. However, I was waiting until having a better implementation. The next release of BioC is in September (I think), so the idea would be to submit in the next couple of months.

Finally, I added you as a collaborator for BASiCS :-)

Best,

Cata

PS: I profiled most of my functions. I realised which ones are the most expensive ones, which against my intuition, were not the ones that depended on the code I borrowed from someone else. Overall, the big bottleneck is that loop we saw yesterday, which is repeated several times throughout the code.

**Comment by Horta** [comment]
If the major bottleneck is that loop you are in business. That should not be hard to parallelize that. I will do a pull request at some point today with that parallelization thing! Thank you!

**Comment by catavallejos** [comment]
That's great news! If it helps, the most expensive updates are for "nu" and "delta" (defined as time/nparameters).

When making the changes, would you mind creating new functions (e.g., muUpdateParallel) rather than modifying the existing ones? I would like to keep the option of running the implementation as it is as well.

**Comment by Horta** [comment]
Sure. It is important to guarantee the we won't break other people's code.

**Pull request merged**